lib/buchungsstreber/parser.rb

```diff
@@ -2,9 +2,10 @@
 class TimesheetParser
   PARSERS = []
 
-  def initialize(file, templates)
+  def initialize(file, templates, minimum_time)
     @file = file
-    @parser = choose_parser(file).new(templates)
+    @minimum_time = minimum_time
+    @parser = choose_parser(file).new(templates, minimum_time)
```

Comment:
Instead of lowering the minimun_time to the parser itself, you could use a local instance variable `@minimum_time`, as the only place that is needed is inside the `TimesheetParser` class.

Reply:
nah..